### ISSUE TITLE
Can this dicomize ECG raw data?

### ISSUE THREAD

**Issue by ppazos** [marcodebe/dicomecg_convert#6]

**Comment by marcodebe** [comment]
No, it actually does the inverse operation: the function ``_signals`` extracts the channels' raw data  from the dicom file.  What you need is a ``_signals⁻¹`` function.
Would you please provide me some sample file of ECG raw data?

**Comment by ppazos** [comment]
@marcodebe yes, let me get some data and get back to you. Thanks!
